### PR TITLE
Add docker_images_list.md

### PR DESCRIPTION
**Pull request by ZePan110**
## Description

The summary of the proposed changes as long as the relevant motivation and context.

## Issues

List the issue or RFC link this PR is working on. If there is no such link, please mark it as `n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

File: docker_images_list.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Example images
```
Comment (chensuyue) [inline docker_images_list.md:1]: Need descriptions for this page.

**Review by chensuyue — APPROVED** [review]

**Comment by kevinintel** [comment]
please give the simple description like difference of chat-qna vs chat-qna-ui

**Comment by chensuyue** [comment]
> please give the simple description like difference of chat-qna vs chat-qna-ui

I think those docker image description should be in docker hub, 
image

File: docker_images_list.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    <td class="tg-yk8o"><a href="https://hub.docker.com/r/opea/chatqna-ui">opea/chatqna-ui</a></td>
+    <td class="tg-na8o">https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/ui/docker/Dockerfile</td>
+  </tr>
+  <tr>
+    <td class="tg-yk8o"><a href="https://hub.docker.com/r/opea/chatqna">opea/chatqna</a></td>
+    <td class="tg-na8o">https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/Dockerfile</td>
+  </tr>
+  <tr>
+    <td class="tg-yk8o"><a href="https://hub.docker.com/r/opea/chatqna-conversation-ui">opea/chatqna-conversation-ui</a></td>
+    <td class="tg-na8o">https://github.com/opea-project/GenAIExamples/blob/add_up_list/ChatQnA/docker/ui/docker/Dockerfile.react</td>
```
Comment (chensuyue) [inline docker_images_list.md:23]: fix the link

File: docker_images_list.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  </tr>
+  <tr>
+    <td class="tg-yk8o"><a href="https://hub.docker.com/r/opea/docsum-ui">opea/docsum-ui</a></td>
+    <td class="tg-yk8o"><a href="https://github.com/opea-project/GenAIExamples/blob/main/DocSum/docker/ui/docker/Dockerfile">https://github.com/opea-project/GenAIExamples/blob/main/DocSum/docker/ui/docker/Dockerfile</a></td>
```
Comment (chensuyue) [inline docker_images_list.md:27]: remove <a>

File: docker_images_list.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    <td class="tg-yk8o"><a href="https://hub.docker.com/r/opea/codegen-ui">opea/codegen-ui</a></td>
+    <td class="tg-na8o">https://github.com/opea-project/GenAIExamples/blob/main/CodeGen/docker/ui/docker/Dockerfile</td>
+  </tr>
+  <tr>
+    <td class="tg-yk8o"><a href="https://hub.docker.com/r/opea/codegen">opea/codegen</a></td>
+    <td class="tg-na8o">https://github.com/opea-project/GenAIExamples/blob/main/CodeGen/docker/Dockerfile</td>
+  </tr>
+  <tr>
+    <td class="tg-yk8o"><a href="https://hub.docker.com/r/opea/searchqna/tags">opea/searchqna</a></td>
+    <td class="tg-yk8o"><a href="https://github.com/opea-project/GenAIExamples/blob/main/SearchQnA/docker/Dockerfile">https://github.com/opea-project/GenAIExamples/blob/main/SearchQnA/docker/Dockerfile</a></td>
```
Comment (chensuyue) [inline docker_images_list.md:51]: remove <a>

File: docker_images_list.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+# Example images
+
+<table class="tg"><thead>
+  <tr>
+    <th class="tg-cly1">Example Images</th>
+    <th class="tg-cly1">Dockerfile</th>
+  </tr></thead>
+<tbody>
+  <tr>
+    <td class="tg-yk8o"><a href="https://hub.docker.com/r/opea/chatqna">opea/chatqna</a></td>
```
Comment (hshen14) [inline docker_images_list.md:16]: add another column to show the short description of the docker image

**Review by kevinintel — APPROVED** [review]

**Pull request merged**